front_end/ui/components/settings/settingCheckbox.css

```diff
@@ -5,6 +5,7 @@
  */
 
 :host {
+  display: block;
```

Comment:
Needed for the existing animation util to work.

The util attempts to animation the background on a Shadow DOM element `<setting-checkbox>`:

https://github.com/facebookexperimental/rn-chrome-devtools-frontend/blob/8de9257f1ca9f129a763165e6df7dac9583f3eb7/front_end/panels/utils/utils.ts#L110

But since the host element is not a `block` by default, the background wouldn't be applied:
https://stackoverflow.com/a/60702911